README.adoc

```diff
@@ -1640,6 +1640,40 @@ Sending emails while generating page response should be avoided.
 It causes delays in loading of the page and request can timeout if multiple email are sent.
 To overcome this emails can be sent in background process with the help of https://github.com/mperham/sidekiq[sidekiq] gem.
 
+=== Enqueuing Many Emails at Once [[enqueuing-many-emails]]
```

Comment:
Is this because there’s no dedicated section for jobs? This equally relates to any jobs?

There’s https://github.com/toptal/active-job-style-guide, but I don’t know who maintains/owns it now.